torch/_library/custom_ops.py

```diff
@@ -760,6 +762,16 @@ def wrapped_func(keyset, *args, **kwargs):
         else:
             return register(func)
 
+    def register_jvp(
```

Comment:
There are some possible APIs:
- we have a register_jvp API
- we merge this in with register_autograd
- we do both
Maybe worth discussing in a larger setting